sdk/storage/azure-storage-blob-stress/src/main/java/com/azure/storage/blob/stress/AppendBlobOutputStream.java

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.stress;
+
+import com.azure.core.util.Context;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.storage.blob.BlobAsyncClient;
+import com.azure.storage.blob.BlobClient;
+import com.azure.storage.blob.specialized.AppendBlobClient;
+import com.azure.storage.blob.specialized.BlobOutputStream;
+import com.azure.storage.blob.stress.utils.OriginalContent;
+import com.azure.storage.stress.CrcInputStream;
+import com.azure.storage.stress.StorageStressOptions;
+import reactor.core.publisher.Mono;
+
+import java.io.IOException;
+
+import static com.azure.core.util.FluxUtil.monoError;
+
+public class AppendBlobOutputStream extends BlobScenarioBase<StorageStressOptions> {
+    ClientLogger LOGGER = new ClientLogger(BlockBlobOutputStream.class);
+    private final OriginalContent originalContent = new OriginalContent();
+    private final BlobClient syncClient;
+    private final BlobAsyncClient asyncNoFaultClient;
+    // this blob is used to perform normal upload in the setup phase
+    private final BlobAsyncClient tempSetupBlobClient;
+
+    public AppendBlobOutputStream(StorageStressOptions options) {
+        super(options);
+        String blobName = generateBlobName();
+        String tempBlobName = generateBlobName();
+
+        this.asyncNoFaultClient = getAsyncContainerClientNoFault().getBlobAsyncClient(blobName);
+        this.syncClient = getSyncContainerClient().getBlobClient(blobName);
+        this.tempSetupBlobClient = getAsyncContainerClientNoFault().getBlobAsyncClient(tempBlobName);
+    }
+
+    @Override
+    protected void runInternal(Context span) throws IOException {
+        AppendBlobClient appendBlobClient = syncClient.getAppendBlobClient();
+
+        BlobOutputStream outputStream = appendBlobClient.getBlobOutputStream(true);
+        try (CrcInputStream inputStream = new CrcInputStream(originalContent.getBlobContentHead(), options.getSize())) {
+            byte[] buffer = new byte[4096]; // Define a buffer
+            int bytesRead;
+
+            // Read from the inputStream and write to the blobOutputStream
+            while ((bytesRead = inputStream.read(buffer)) != -1) {
+                outputStream.write(buffer, 0, bytesRead);
+            }
+
+            // Ensure to close the blobOutputStream to flush any remaining data and finalize the blob.
+            outputStream.close();
```

Comment:
nit: Use the `BlobOutputStream` as part of the try-with-resources block

```java
        try (CrcInputStream inputStream = new CrcInputStream(originalContent.getBlobContentHead(), options.getSize());
             BlobOutputStream outputStream = appendBlobClient.getBlobOutputStream(true)) {
```

Reply:
Need one more change here to remove this close as the try-with-resource will handle that. Unless this close was being called to perform flush, if so, call flush here instead of close